Description: deactivating assertions that appear to be flaky
Author: Pierre Gruet <pgt@debian.org>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1031480
Forwarded: https://github.com/EsotericSoftware/reflectasm/issues/91
Last-Update: 2023-02-17

--- a/test/com/esotericsoftware/reflectasm/ClassLoaderTest.java
+++ b/test/com/esotericsoftware/reflectasm/ClassLoaderTest.java
@@ -60,7 +60,7 @@
 		assertEquals(access1.getClass().toString(), access2.getClass().toString()); // Same class names
 		assertFalse(access1.getClass().equals(access2.getClass())); // But different classes
 
-		assertEquals(initialCount + 2, AccessClassLoader.activeAccessClassLoaders());
+		//assertEquals(initialCount + 2, AccessClassLoader.activeAccessClassLoaders());
 
 		testClassLoader1 = null;
 		testClass1 = null;
@@ -74,7 +74,7 @@
 		reclaimLoaders();
 
 		// Yeah, reclaimed!
-		assertEquals(initialCount, AccessClassLoader.activeAccessClassLoaders());
+		//assertEquals(initialCount, AccessClassLoader.activeAccessClassLoaders());
 	}
 
 	private void reclaimLoaders () throws Exception {
